prisonstaffhub-specs/src/test/groovy/uk/gov/justice/digital/hmpps/prisonstaffhub/specs/AgencySelectionSpecification.groovy

```diff
@@ -45,8 +48,89 @@ class AgencySelectionSpecification extends BrowserReportingSpec {
         elite2api.stubUpdateActiveCaseload()
         notmServer.stubFor(
                 get(WireMock.urlPathMatching('/.*'))
-                .willReturn(
-                    aResponse().withStatus(200)))
+                        .willReturn(
+                                aResponse().withStatus(200)))
+
+        $('#info-wrapper').click()
```

Comment:
Low-level details like element names and ids should really reside in the page objects.

Reply:
Yeah, I think these aren't part of a page because they are the header. I'll have a think about how to fix those and follow-up.